ontology/yaml/resources/LIGHTING/entity_types/ABSTRACT.yaml

```diff
@@ -30,6 +30,18 @@ DC:
   uses:
   - brightness_percentage_command
 
+LC_AWIC:
```

Comment:
Please remove the `LC_` so the type name is simply `AWIC` (the underscore is what makes this type name look confusing)

Reply:
...done